publishing.md

```diff
@@ -53,8 +53,12 @@ Please make sure to update the destination of the source code link if you use a
 </span>
 ```
 
-## Changes
+## Distributing changes
```

Comment:
To me only the last added section is adding value.

I would suggest another addition stating something like:
> You see opportunities in developping services/business/tools on top of OpenFisca but are concerned with the AGPL3 licence implications. Please create an issue to have an open discussion or contact us by email

Reply:
Thanks @guillett.

When you say _adding value_, you mean in terms of the objectives of the pull request as stated in the description or is it based on another value reference ?

> You see opportunities in developing services/business/tools on top of OpenFisca but are concerned with the AGPL3 licence implications. Please create an issue to have an open discussion

That's great, I'll add a phrase to that end! 😄